Rdmp.Core/Databases/CatalogueDatabase/up/078_AddLastLoadTimeToLoadMetadata.sql

```diff
@@ -0,0 +1,7 @@
+﻿--Version:8.1.3
+--Description: Adds LastLoadTime to LoadMetadata table
+ GO
+  if exists (select 1 from sys.columns where name = 'LoadLastTime' and object_id = (select object_id from sys.tables where name = 'LoadMetadata'))
```

Comment:
Shouldn't this be 'if _not_ exists', to add the column only if it _isn't_ already present...?